packages/next-swc/crates/next-dev-tests/tests-manifest.js

```diff
@@ -0,0 +1,729 @@
+// Tests that are currently enabled with Turbopack in CI.
+// Add new test when Turbopack updates to fix / implement a feature.
+const enabledTests = [
+  'test/development/acceptance-app/dynamic-error.test.ts',
+  'test/development/basic/legacy-decorators.test.ts',
+  'test/integration/plugin-mdx-rs/test/index.test.js',
+  'test/e2e/app-dir/underscore-ignore-app-paths/underscore-ignore-app-paths.test.ts',
+  'test/e2e/postcss-config-cjs/index.test.ts',
+]
+
+// Tests that are currently disabled with Turbopack in CI.
+const disabledTests = [
```

Comment:
How is that used? Seems unused...?

Reply:
It is intentionally reused for now. Currently more for easily listing out all of the test updated, since keeping it in spreadsheet was tricky to sync to the latest.